yaml.lint.cloudbuild.yaml

```diff
@@ -4,9 +4,17 @@ steps:
   entrypoint: '/bin/sh'
   args:
   - -c
-  - | # Recursively find yaml files (ignored files in .yamllint config)
-    find ./* -name '*.yaml' -not -path "./testing/*" \
-      -exec yamllint -f parsable {} \; \
-      | tee /proc/1/fd/1 \
-      | grep -q 'error'
-    if [[ $? -eq 0 ]] ; then exit 1 ; fi
+  - | # Recursively find yaml files (yamllint must be run in directory)
+    find . -name "*.yaml" -not -path "./testing/*" -not -path "*.lint.cloudbuild.yaml" \
+      -exec sh -c '
+      root=$(pwd)
+      lint () {
+        cd "$root" || exit
```

Comment:
pushd and popd won't work in the shell environment

Reply:
That's surprising. Maybe it would require `bash`? Fine to keep `cd`.

Reply:
It does require bash but I don't believe that comes installed in the alpine image.